tests/dind.sh

```diff
@@ -1,15 +1,32 @@
-#!/bin/sh
+#!/bin/bash
 
+K3D_EXE=${EXE:-/bin/k3d}
 K3D_IMAGE_TAG=$1
 
-k3de2e=$(docker run -d --rm \
+# define E2E_KEEP to non-empty for keeping the e2e runner container after running the tests
+E2E_KEEP=${E2E_KEEP:-}
+
+####################################################################################
+
+TIMESTAMP=$(date "+%H%M%S")
```

Comment:
Maybe we should add the date here as well just to be really sure?

Reply:
Added month and day.